content/telegraf/v1.24/configure_plugins/custom_builder.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reduce Telegraf binary size with customized agent
+description: Use the custom builder tool to reduce Telegraf binary disk space. 
+menu:
+  telegraf_1_24:
+
+    name: Reduce binary size
+    Parent: Configure plugins
+    weight: 20
+---
+Reduce Telegraf's overall memory and disk footprint with the custom builder. This tool allows you to select which plugins to use in the Telegraf binary based on your configuration file. 
+
+## Requirements
+
+- Ensure you've installed the current version of [Go](https://go.dev/). 
+- Create your Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1.24/configuration/).
+
+## Build and run the custom builder
+
+1. Clone the Telegraf repository:
+    ```
+    git clone https://github.com/influxdata/telegraf.git
+    ```
+2. Change directories into the top-level of the Telegraf repository:
+    ```
+    cd telegraf
+    ```
+3. Build the Telegraf custom builder tool by entering the folllowing command:
+    ```
+    make build_tools
+    ```
+4. Point the custom builder at your Telegraf configuration file or files that include the plugins you want to build, like in the following examples, to generate your customized binary:
```

Comment:
```suggestion
4. Run the `custom_builder` utility with at least one `--config` or `--config-directory` flag to specify Telegraf configuration files to build from. `--config` accepts local file paths and URLs. `--config-dir` accepts local directory paths. You can include multiple `--config` and `--config-dir` flags. The custom builder builds a `telegraf` binary with only the plugins included in the specified configuration files or directories:
```

Reply:
To specify Telegraf configuration files to build from, run `custom_builder` with one of the following:
- `--config` to accept local file paths and URLs
- `--config-dir` to accept local directory paths
**Note:** Use multiple `--config` and `--config-dir` to use multiple configs?

The custom builder builds a `telegraf` binary with the specified plugins in the specified configuration files or directories.

Reply:
should we say using the specified configuration files....or is the term "in" accurate here?